src/DssSpell.t.sol

```diff
@@ -496,4 +439,30 @@ contract DssSpellTest is DssSpellTestBase {
         }
         assertEq(expectedHash, actualHash);
     }
+
+    function test_flashKiller() public {
```

Comment:
I would also add a `testFail` to confirm that the FlashKiller can't be used prior to shutdown, just for completeness.